AmahiAnywhere/AmahiAnywhere/Presentation/Files/FilesViewController+UITableViewDelegates.swift

```diff
@@ -26,6 +27,21 @@ extension FilesViewController : UITableViewDelegate, UITableViewDataSource {
             cell.fileSizeLabel?.text = serverFile.getFileSize()
             cell.lastModifiedLabel?.text = serverFile.getLastModifiedDate()
             
+            var imageName = ""
+            if (serverFile.mime_type?.starts(with: "image"))! {
+                imageName = "image"
+            }
+            else if (serverFile.mime_type?.starts(with: "audio"))! {
+                imageName = "audio"
+            }
+            else if (serverFile.mime_type?.starts(with: "video"))! {
+                imageName = "video"
+            }
+            else {
+                imageName = "file"
+            }
+            cell.thumbnailImage.sd_setImage(with: URL(string: ServerApi.shared!.getFileUri(serverFile).absoluteString), placeholderImage: UIImage(named: imageName))
```

Comment:
Should use the `options:SDWebImageRefreshCached` to respect caching headers sent by server. Read the `Handle image refresh` section in the docs [here](http://cocoadocs.org/docsets/SDWebImage/4.1.2/).
For swift check this [answer](https://stackoverflow.com/a/43814202/6352482).